src/utils/Device.ts

```diff
@@ -0,0 +1,24 @@
+interface DeviceInfo {
+    id: string;
+    model: string;
+    os: number;
+    inputMode: number;
+    guiScale: number;
+}
+
+class Device {
+    public inputMode: number;
+    public id: string;
+    public os: number;
+    public model: string;
+    public guiScale: number;
+
+    constructor(options: DeviceInfo) {
```

Comment:
I think we should consider passing these as parameters in the constructor. I understand that it's based on how its written in the packet stream, but I feel as if this is repetitive, thoughts? @filfat @HerryYT